### PR TITLE
Bug fixes for listing/removing images.

### PR DESCRIPTION
**Pull request by Alexhuszagh**
Does nothing if there are no images to remove, and now correctly lists the old rustembedded images.

### CODE REVIEW

File: src/bin/commands/images.rs [hunk 1]
```diff
@@ -185,7 +195,9 @@ fn remove_images(
         command.arg("--force");
     }
     command.args(images);
-    if execute {
+    if images.is_empty() {
+        Ok(())
+    } else if execute {
         command.run(verbose, false).map_err(Into::into)
     } else {
         println!("{:?}", command);
```
Comment (Emilgardis) [inline src/bin/commands/images.rs:203]: this can use the new `command_pretty()` fn

**Comment by Alexhuszagh** [comment]
bors r-
bors r=Emilgardis

File: src/bin/commands/images.rs (hunk 1 above)
Comment (Emilgardis) [reply src/bin/commands/images.rs:203]: i don't see this being done, but it's fine

Comment (Alexhuszagh) [reply src/bin/commands/images.rs:203]: `Command::run` invokes `Command::run_and_get_status` which invokes `Command::print_verbose` which then called `Command::command_pretty`.

Comment (Alexhuszagh) [reply src/bin/commands/images.rs:203]: Oh.... Oh.... Nevermind. I'll cancel this.

**Comment by Alexhuszagh** [comment]
bors r=Emilgardis

**Pull request merged**